src/routes/index.tsx

```diff
@@ -35,10 +35,19 @@ import {
 } from './constants';
 
 const PrivateRoute = ({ children }: any) => {
-  const isOnboarded = useTypedSelector(
+  const isFullyOnboarded = useTypedSelector(
     ({ settings }) => !!(settings.macaroonCredentials && settings.tdexdConnectUrl)
   );
-  return isOnboarded ? children : <Navigate to={ONBOARDING_PAIRING_ROUTE} />;
+  const isPairedWithNotInitDaemon = useTypedSelector(
+    ({ settings }) => !!(!settings.macaroonCredentials && settings.tdexdConnectUrl)
+  );
+  return isFullyOnboarded ? (
+    children
+  ) : isPairedWithNotInitDaemon ? (
```

Comment:
what happens with a TDEX Daemon insecure without macaroon? Should not happens, but user can decide to run it NO_MACAROONS=true.

I think we should fetch this from isReady RPC (since we have the URL)